src/jit/codegencommon.cpp

```diff
@@ -3981,10 +4001,12 @@ void            CodeGen::genFnPrologCalleeRegArgs(regNumber xtraReg,
         {
             // A struct might be passed  partially in XMM register for System V calls.
             // So a single arg might use both register files.
+#ifndef ARM_SOFTFP
             if (isFloatRegType(regType) != doingFloat)
```

Comment:
Yes, the incoming register arguments will be spilled to a stack location assigned to the variable.